x-pack/filebeat/input/streaming/input_test.go

```diff
@@ -128,7 +128,8 @@ var inputTests = []struct {
 					"events": [inner_body],
 				})`,
 		},
-		response: []string{`
+		response: []string{
```

Comment:
probably caused by changing from gofmt to gofumpt and enabling auto format on save.